packages/client3/components/design-system/file-upload/file-upload.tsx

```diff
@@ -0,0 +1,65 @@
+import clsx from "clsx";
+import React, { forwardRef } from "react";
+import { useFormContext } from "react-hook-form";
+
+import { Icon } from "../icon";
+import { Caption } from "../typography";
+
+interface FileUploadProps {
+  name: string;
+  id?: string;
+  errorMessage?: string;
+  disabled?: boolean;
+}
+
+export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
+  function FileUpload({ name, errorMessage, disabled }, ref) {
+    const formContext = useFormContext();
+    let _errorMessage = errorMessage;
+    if (formContext !== null) {
+      _errorMessage = formContext.formState.errors[name]?.message;
+    }
+    const isError = !!_errorMessage;
+
+    return (
+      <>
+        <div className="flex w-full items-center justify-center">
+          <label
+            htmlFor="dropzone-file"
+            className={clsx(
+              "flex h-64 w-full cursor-pointer flex-col items-center justify-center rounded-lg bg-dark-90 hover:bg-dark-80",
+              "border border-dashed",
+              isError ? "border-state-error" : "border-dark-70",
+              disabled && "opacity-50"
+            )}
+          >
+            <div className="flex flex-col items-center justify-center pt-5 pb-6">
+              <Icon
+                name={"AddCircle"}
+                size="md"
+                className={clsx("my-[10px]")}
```

Comment:
10px is 2.5 in tailwind's mapping. So this could be replaced to my-2.5

Reply:
Lol thanks :P, I've started just going with the pixels because its easier to reference from the figma.